test/EcdsaXi2023Cryptosuite.spec.js

```diff
@@ -203,8 +309,34 @@ describe('EcdsaXi2023Cryptosuite', () => {
     it('should fail to sign with non-bytes extraInformation', async () => {
       const unsignedCredential = JSON.parse(JSON.stringify(credential));
       unsignedCredential.type.push('UndefinedType');
+      const badXI = 100;
+      const badCryptosuite = createCryptosuite({badXI});
+      const keyPair = await EcdsaMultikey.from({...ecdsaMultikeyKeyPair});
+      const date = '2023-03-01T21:29:24Z';
+      const suite = new DataIntegrityProof({
+        signer: keyPair.signer(), date, cryptosuite: badCryptosuite
+      });
+
+      let error;
+      try {
+        await jsigs.sign(unsignedCredential, {
+          suite,
+          purpose: new AssertionProofPurpose(),
+          documentLoader
+        });
+      } catch(e) {
+        error = e;
+      }
+
+      expect(error).to.exist;
+      expect(error.name).to.equal('jsonld.ValidationError');
+    });
+
+    it('should fail to sign with no extraInformation', async () => {
+      const unsignedCredential = JSON.parse(JSON.stringify(credential));
+      unsignedCredential.type.push('UndefinedType');
```

Comment:
- Remove. Same issue as above.  That was causing the safe mode error to get set and checked below..
- Is this needed?  `{}` is empty options to the API, which will set the default XI.  So the test passes.

Reply:
Good point. The code that gets mad about no extraInformation is in the issuer, not here. This test does nothing. Deleted.